examples/nextjs-14-app-dir-si/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "nextjs-14-app-dir-rl",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@arcjet/env": "file:../../env",
```

Comment:
Why is this being pulled in?

Reply:
Good question, I copied the deps from nextjs-14-app-dir-rl and it was there. But doesn't seem to be needed.

Reply:
We should probably remove it from that package too, but out of scope for this PR

Reply:
Please open an issue or PR that cleans this up

Reply:
#1400